superset-frontend/plugins/legacy-preset-chart-deckgl/src/layers/Scatter/Scatter.jsx

```diff
@@ -36,14 +36,11 @@ function setTooltipContent(formData, verboseMap) {
     return (
       <div className="deckgl-tooltip">
         <TooltipRow
-          label={`${t('Longitude and Latitude')}: `}
```

Comment:
I could see endless debate about translating colons and spaces... I'm not sure if I have strong feelings, but I can't help but wonder if strings like "Longitude and Latitude" might be recycled elsewhere, whereas they're not as reusable with the `: ` appended.